recipes/jupyter-dashboards/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.6.0" %}
+{% set sha256 = "cb198c9e85e259dbdd34c3dbdaf97e9c63fa6c7ee86caa5bbf719859a799f3db" %}
+{% set name = "jupyter_dashboards" %}
+
+package:
+  name: {{ name }} 
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - {{ name.replace('_', '-') }} = jupyter_dashboards.extensionapp:main 
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - notebook
+    - python
+    - traitlets
+
+test:
+  imports:
+    - {{ name }} 
+  commands:
+    - {{ name.replace('_', '-') }} --help
+    - jupyter dashboards --help
+
+about:
+  home: http://github.com/jupyter-incubator/dashboards
+  license: BSD 3-Clause 
+  # pypi package is busted: does not include LICENSE.md! will be fixed in 0.6.1
+  # license_file: LICENSE.md
```

Comment:
This PR ( https://github.com/jupyter-incubator/dashboards/pull/274 ) should help.

We could also package from source instead of PyPI to have access to this file.

Some people include the license file with the recipe. Then it is merely a matter of copying it into the source directory during the build phase.